src/routes/student.js

```diff
@@ -147,4 +147,40 @@ router.route('/add').post(async (req, res) => {
 
 });
 
+/**
+ * Get student studyhours
+ * @param {string} email Student email
+ * @return {number} Total number of studyhours 
+ */
+router.route('/studyhours/:email').get((req, res) => {
+    StudentRepository.findOneByEmail(req.params.email)
+        .then((student) => {
+            let courses = student.courses;
+            let totalHours = 0;
+            for (let course of courses) {
+                totalHours += parseFloat(course.classUnit) * 1.5;
```

Comment:
No need to calculate the study hours per course before adding to the total study hours. Note that adding a course event already adds a `studyHours` attribute. So `totalHours += parseFloat(course.studyHours);` should be enough.

Reply:
This is equivalent, I just copy pasted the hours calculation from retrieving single courses.

Reply:
I understand. Was just thinking about the point of `studyHours` attribute if we're going to keep recalculating it anywhere it's needed. It's redundant.